src/clj/rems/db/applications.clj

```diff
@@ -100,6 +100,10 @@
 (mount/defstate application-cache
   :start (atom (cache/ttl-cache-factory {} :ttl 10000)))
 
+;; TODO combine with reload-cache!?
+(defn reset-application-cache! []
+  (swap! application-cache empty))
```

Comment:
Yes, better combine with `reload-cache!`.

Reply:
I'll do this